android/app/src/test/java/com/cras/app/ui/inbox/InboxViewModelAccountLifecycleTest.kt

```
@@ -0,0 +1,374 @@
+    private class FakeCommentService : CommentService {
+        override suspend fun fetchComments(session: OperatorSession, taskId: String?) = emptyList<Comment>()
+        override suspend fun createComment(session: OperatorSession, params: com.cras.app.data.CreateCommentParams) = throw NotImplementedError()
+    }
+
+    private class FakeVoiceRecordingStore : VoiceRecordingStore {
+        var clearCount = 0
+        override fun save(wav: ByteArray, createdAtEpochMs: Long) = throw NotImplementedError()
+        override fun list() = emptyList<com.cras.app.voice.RetainedRecording>()
+        override fun latest() = null
+        override fun readBytes(id: String) = null
+        override fun delete(id: String) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)